main.go

```diff
@@ -18,50 +18,60 @@ package main
 
 import (
 	"flag"
-	"os"
-	"strings"
-
 	"github.com/kiegroup/kogito-operator/core/client"
 	"github.com/kiegroup/kogito-operator/core/logger"
 	"github.com/kiegroup/kogito-operator/meta"
-	"k8s.io/apimachinery/pkg/runtime"
+	"os"
+	"strings"
+
+	// Import all Kubernetes client auth plugins (e.g. Azure, GCP, OIDC, etc.)
+	// to ensure that exec-entrypoint and run can make use of them.
+	_ "k8s.io/client-go/plugin/pkg/client/auth"
 
-	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+	"k8s.io/apimachinery/pkg/runtime"
```

Comment:
shouldn't it be replaced by `sigs.k8s.io/controller-runtime/pkg/client` ?

Reply:
`Scheme` is available in `"k8s.io/apimachinery/pkg/runtime"` only.